spring-amqp/src/test/java/org/springframework/amqp/support/converter/Jackson2XmlMessageConverterTests.java

```diff
@@ -0,0 +1,381 @@
+/*
+ * Copyright 2002-2017 the original author or authors.
```

Comment:
Please, update Copyright in all the affected classes to the `2018` in the last number, but all the new classes must have only `2018`.
Also don't forget to add `@since 2.1` to new classes.